README.md

```diff
@@ -32,7 +32,7 @@
 * ["Entry not found in cache" error when importing ACR and ASA modules](https://github.com/Microsoft/vscode-azure-iot-edge/issues/161)
 * [Node.js module template version is not applied when another version is installed globally](https://github.com/Microsoft/vscode-azure-iot-edge/issues/340)
 
-## Prerequisites
+## Prerequisites to make Azure IoT Edge extension work
```

Comment:
Prerequisites for Azure IoT Edge extension?

Reply:
OK, to make it align